components/script/dom/resizeobserver.rs

```diff
@@ -315,7 +315,16 @@ fn calculate_box_size(target: &Element, observed_box: &ResizeObserverBoxOptions)
                 .pop()
                 .unwrap_or_else(Rect::zero)
         },
```

Comment:
https://github.com/servo/servo/blob/9da9eed8f0f7e2ea9b02a930f402b04776e2e256/components/script/dom/resizeobserver.rs#L309-L317

Maybe we should use content boxes for these as well. Maybe we could use `Node::content_box` that calculates the bounding box instead of taking the sizes of the first box fragment which is I believe what are we doing here. 

https://drafts.csswg.org/resize-observer/#calculate-box-size-h
> The sizes of these elements are always their bounding box sizes, because these elements do not use standard CSS box model.